src/tests/ftest/launch.py

```diff
@@ -1182,15 +1177,23 @@ def report_skipped_test(test_file, avocado_logs_dir, reason):
     """
     message = "The {} test was skipped due to {}".format(test_file, reason)
     print(message)
+
+    # Generate a fake avocado results.xml file to report the skipped test.
+    # This file currently requires being placed in a job-* subdirectory.
     test_name = get_test_category(test_file)
-    destination = os.path.join(avocado_logs_dir, test_name)
-    os.mkdir(destination)
+    time_stamp = datetime.now().strftime("%Y-%m-%dT%H.%M")
+    destination = os.path.join(
+        avocado_logs_dir, "job-{}-da03911-{}".format(time_stamp, test_name))
+    try:
+        os.makedirs(destination)
+    except (OSError, FileExistsError) as error:
+        print("Failed to create {}: {}".format(destination, error))
```

Comment:
Do we need to print this?  Just starting to get sensitive to the amount of output log processing is generating now.  We seem to print a lot of "things are good" type logging which is burying the "things are bad" needles in the haystack.

Reply:
We really should never hit this error and so I would expect this to be extremely infrequent.  If we are calling this method we are skipping running `avocado run ...` so there should no existing subdirectory for this test.  So I would consider the existence of the directory a possible concern.  That said we shouldn't be overwriting any files unless a `framework_results.xml` file exists in the existing directory - hence why we just print the message and continue.

Reply:
Fair point.

Reply:
Yes, agreed.  A clearer explanation is almost always better.